deployment/terraform/network.tf

```diff
@@ -1,7 +1,7 @@
 module "vpc" {
   source = "github.com/azavea/terraform-aws-vpc?ref=6.0.1"
 
-  name                       = "vpc${replace(var.project, " ", "")}${var.environment}"
+  name                       = "vpc${replace(var.project, " ", "")}"
   region                     = var.aws_region
   key_name                   = var.aws_key_name
```

Comment:
The PEM key used for the bastion has been recreated and placed in the fileshare. The old staging PEM has been removed.